CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV changelog](https://github.com/pingcap/tikv/blob/master/CHANGELOG.md) and [PD changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.0.5] - 2018-07-06
+### New Features
+  - Add the `tidb_disable_txn_auto_retry` system variable, used to disable the automatic retry of transactions #6877
```

Comment:
Do we need to change the plain test `#6877` to a link  [#6867](https://github.com/pingcap/tidb/pull/6877). Or just remove it?

Reply:
"which is used to disable xxx"?